apps/books/templates/home.html

```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
+{% block content %}
+<ul style="list-style-type: none;">
+    {% for book in books %}
+    {% if book.is_featured %}
```

Comment:
No need to check it here .. Please filter the books on backend and pass only featured books in context .. 
Secondly use Managers to write a filter query

Reply:
Secondly use Managers to write a filter query???